.github/CODEOWNERS

```diff
@@ -116,29 +116,30 @@ common/_themeOverrides.scss @microsoft/cxe-red @phkuo
 common/_common.scss @microsoft/cxe-red @phkuo
 
 ## Component packages
-packages/react-avatar/ @microsoft/cxe-red @behowell
-packages/react-menu/ @microsoft/teams-prg
 packages/react-accordion/ @microsoft/teams-prg
+packages/react-avatar/ @microsoft/cxe-red @behowell @khmakoto
 packages/react-badge/ @ling1726 @layershifter @assuncaocharles @behowell
 packages/react-button/ @microsoft/cxe-red @dzearing @khmakoto
 packages/react-card/ @microsoft/cxe-prg
 packages/react-checkbox/ @microsoft/cxe-red @khmakoto
 packages/react-components/ @microsoft/teams-prg @microsoft/cxe-prg @microsoft/cxe-red
+packages/react-divider/ @microsoft/cxe-prg
 packages/react-focus/ @microsoft/cxe-red @khmakoto
 packages/react-image/ @microsoft/cxe-prg
-packages/react-text/ @microsoft/cxe-prg
 packages/react-input/ @microsoft/cxe-red @ecraig12345
+packages/react-label/ @microsoft/cxe-red
+packages/react-link/ @microsoft/cxe-red @khmakoto
+packages/react-menu/ @microsoft/teams-prg
+packages/react-slider/ @microsoft/cxe-red
 packages/react-storybook/ @microsoft/cxe-prg @microsoft/teams-prg
 packages/react-storybook-addon/ @microsoft/cxe-prg
 packages/storybook/ @microsoft/cxe-prg @microsoft/teams-prg
```

Comment:
I feel like these should go into the regular `Packages` section in line 74 above since they are not dealing with components in the library.

Reply:
I thought this sections was all the parts that are shipping as part of the beta because they are referenced by react-components.  What is meant by 'Component Packages' then?

Reply:
Good point. :)